src/rdiffbackup/run.py

```diff
@@ -23,6 +23,18 @@
 from rdiff_backup import Globals, log
 from rdiffbackup import arguments, actions_mgr, actions
 
+try:
+    import msvcrt
+
+    msvcrt.setmode(0, os.O_BINARY)
```

Comment:
A comment would be helpful to understand why it's needed

Reply:
To be honest, I took over the code as it stands, and never cared as it worked. But, good point, and clarified.